actionpack/test/dispatch/mime_type_test.rb

```diff
@@ -161,8 +161,8 @@ class MimeTypeTest < ActiveSupport::TestCase
   end
 
   test "type should be equal to symbol" do
-    assert_equal Mime[:html], "application/xhtml+xml"
-    assert_equal Mime[:html], :html
+    assert_equal("application/xhtml+xml", Mime[:html])
+    assert_equal(:html, Mime[:html])
```

Comment:
There is a [test failure](https://buildkite.com/rails/rails/builds/99592#018a8888-7ec7-4eca-be4d-15c20e7eebc4/1174-1184) due to this change.

This test is actually testing `Mime::Type#==`, so let's invoke that directly:

```suggestion
    assert_operator Mime[:html], :==, "application/xhtml+xml"
    assert_operator Mime[:html], :==, :html
```